react-stopwatch/my-app/src/App.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+import './App.css';
+import { Stopwatch } from './Stopwatch';
+
+function App() {
+  const [count, setCount] = useState(0);
+  const [pause, setPause] = useState(true);
```

Comment:
Since you can calculate this from other state, it would be best to not have it as state but just calculate it into a local variable.